src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,261 @@
+"""Use DNS twist to fuzz domain names and cross check with a blacklist."""
+# Standard Python Libraries
+import datetime
+import json
+import pathlib
+import traceback
+
+# Third-Party Libraries
+import dnstwist
+import dshield
+import psycopg2.extras as extras
+import requests
+
+# cisagov Libraries
+from pe_reports import app
+
+from .data.pe_db.db_query_source import (
+    addSubdomain,
+    connect,
+    getDataSource,
+    getSubdomain,
+    org_root_domains,
+    query_orgs_rev,
+)
+
+date = datetime.datetime.now().strftime("%Y-%m-%d")
+LOGGER = app.config["LOGGER"]
+
+
+def checkBlocklist(dom, sub_domain_uid, source_uid, pe_org_uid, perm_list):
+    """Cross reference the dnstwist results with DShield Blocklist."""
+    malicious = False
+    attacks = 0
+    reports = 0
+    if "original" in dom["fuzzer"]:
+        return None, perm_list
+    elif "dns_a" not in dom:
+        return None, perm_list
+    else:
+        if str(dom["dns_a"][0]) == "!ServFail":
+            return None, perm_list
+        LOGGER.info(str(dom["dns_a"][0]))
+
+        # Check IP in Blocklist API
+        response = requests.get(
+            "http://api.blocklist.de/api.php?ip=" + str(dom["dns_a"][0])
+        ).content
+
+        if str(response) != "b'attacks: 0<br />reports: 0<br />'":
+            malicious = True
+            attacks = int(str(response).split("attacks: ")[1].split("<")[0])
+            reports = int(str(response).split("reports: ")[1].split("<")[0])
+
+        # Check IP in DSheild API
+        try:
+            results = dshield.ip(str(dom["dns_a"][0]), return_format=dshield.JSON)
+            results = json.loads(results)
+            threats = results["ip"]["threatfeeds"]
+            attacks = results["ip"]["attacks"]
+            attacks = int(0 if attacks is None else attacks)
+            malicious = True
+            dshield_attacks = attacks
```

Comment:
I think this is a bit more Pythonic and cleaner:
```suggestion
            dshield_attacks = int(results["ip"].get("attacks", 0))
            malicious = True
```

I believe this also fixes a bug where the `attacks` variable is being unintentionally overwritten here by the DShield results.  Based on code later, it's my understanding that `attacks` is meant to contain the number of attacks reported via blocklist.de.